snapcraft/plugins/rust.py

```diff
@@ -152,6 +152,13 @@ def _fetch_rust(self):
                   '--disable-sudo', '--save'] + options)
 
     def _fetch_deps(self):
+        
+        source_subdir = getattr(self.options, 'source_subdir', None)
```

Comment:
I can't really figure out why this isn't working; but perhaps this can be a later code change
it's saying that source_subdir is not defined.
and AttributeError: 'Options' object has no attribute 'source_subdir'